### PR TITLE
Replace hand-written wrappers.proto messages.

### PR DESCRIPTION
**Pull request by allevato**
All tests currently pass with this change.

Part of #13.

### CODE REVIEW

File: SwiftProtobufRuntime.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -262,15 +261,16 @@
...
+		AAEA526E1DA6EF83003F318F /* Google_Protobuf_Struct.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Google_Protobuf_Struct.swift; sourceTree = "<group>"; };
 		__PBXFileRef_Package.swift /* Package.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Package.swift; sourceTree = "<group>"; };
 		__PBXFileRef_ProtobufTestSuite_Info.plist /* ProtobufTestSuite_Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = ProtobufTestSuite_Info.plist; path = SwiftProtobufRuntime.xcodeproj/ProtobufTestSuite_Info.plist; sourceTree = SOURCE_ROOT; };
 		__PBXFileRef_Protobuf_Info.plist /* Protobuf_Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = Protobuf_Info.plist; path = SwiftProtobufRuntime.xcodeproj/Protobuf_Info.plist; sourceTree = SOURCE_ROOT; };
-		__PBXFileRef_Sources/Protobuf/Google_Protobuf_Any.swift /* Google_Protobuf_Any.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Google_Protobuf_Any.swift; sourceTree = "<group>"; };
```
Comment (thomasvl) [inline SwiftProtobufRuntime.xcodeproj/project.pbxproj]: Did you mean for Any and Struct to also get removed?

Comment (allevato) [inline SwiftProtobufRuntime.xcodeproj/project.pbxproj]: Thanks for catching that. I added them back (I tried replacing all the WKTs at once, then realized that was a bad idea) but it caused some churn in the pbxproj. Pushed a commit to remove the churn.

**Comment by tbkka** [comment]
I'm curious:  What did this do to the overall size of the compiled module?

I recall that my original thinking behind basing these on a common generic implementation was in the hopes it would reduce the overall code size.

**Comment by allevato** [comment]
Before this commit, the release version of libSwiftProtobuf.dylib on my system was 3,005,612 bytes. After this commit, it's 3,035,608 bytes. So there's a small increase, but it would better to get that savings back elsewhere because maintaining the hand-generated protos will be a huge pain going forward (especially right now, as we make changes to the API).

Once we get CI set up, seeing reports of the dylib size delta for each PR would be *amazing*.

**Pull request merged**